redbadger/continous-compliance
# Serialise JSON files on compliance folder as txt

closes #41 

## How to test

1. Go to `count-dracula` code and pull recent remote branches
```sh
git fetch
```
2. Go to branch `test-continous-compliance-v08`
```sh
git checkout test-continous-compliance-v08
```
3. Do an empty commit and push to remote
```sh
git commit --allow-empty -m "Trigger continous-compliance CI/CD"
git push
```
4. Go to count dracula [actions](https://github.com/redbadger/count-dracula/actions)
5. Wait until `continous compliance` step on **continous-compliance** job finished and open the job.
6. You should see a message like this
```txt
Create text file /home/runner/work/count-dracula/count-dracula/compliance/github/info.txt
Create text file /home/runner/work/count-dracula/count-dracula/compliance/test-results/unit-tests/results.txt
```
7. Click the URL on the last line and should trigger a file download
8. open `test-results` sub folder and check of a file `results.txt`
8. 🚀 
![](https://media.giphy.com/media/zSiEygkcONzyw/giphy.gif)
